src/test/java/victor_murashev/junit/neonStream/NeonStreamSmokeTest.java

```diff
@@ -0,0 +1,85 @@
+package victor_murashev.junit.neonStream;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.interactions.Actions;
+import victor_murashev.ui.neonStream.components.HeroCarousel;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.urlContains;
+
+public class NeonStreamSmokeTest extends NeonStreamBaseTest {
+
+    HeroCarousel herosCarousel = neonStreamPage.heroCarousel;
+    Actions action = new Actions(driver);
+
+
+    @BeforeEach
+    public void placeholdersShouldNotSeen() {
+        actions.shouldNotSee(herosCarousel.placeholder);
+        actions.shouldSee(herosCarousel);
+    }
+
+
+    @Test
+    @DisplayName("Content is displayed")
+    public void contentIsDisplayed() throws InterruptedException {
```

Comment:
It is unacceptably that test method throw exception

Reply:
@dgrimaliuc As I said before, please do not review this pull request. :) 

I put Thread.sleep because of I am getting exceptions in spite of we have BeforeEach method that should avoid them: 

    @BeforeEach
    public void placeholdersShouldNotSeen() {
        actions.shouldNotSee(herosCarousel.placeholder);
        actions.shouldSee(herosCarousel);
    }

Before I understand why this method does work as expected, I put the Thread.sleep. 

BTW, this is one of the reason why I do not like Selenium where we have to create separate methods to wait until POM will be uploaded. In the Playwright it just resolved by using async/await and tester can concentrate his efforts on creating new test cases instead of loosing time on resolving problems like this one.

Reply:
review xpath
for me waating for placeholders to disappear works without timeouts